packages/v4/patternfly-docs/patternfly-docs.source.js

```diff
@@ -54,24 +56,30 @@ module.exports = (sourceMD, sourceProps) => {
   sourceProps(path.join(reactCodeEditorPath, '/**/*.tsx'),reactPropsIgnore);
   sourceProps(path.join(reactChartsPath, '/**/*.tsx'),reactPropsIgnore);
   sourceProps(path.join(reactLogViewerPath, '/**/*.tsx'), reactPropsIgnore);
+  //sourceProps(path.join(reactDragDropPath, '/**/*.tsx'), reactPropsIgnore);
```

Comment:
Maybe for now, unless we want to add TODO comments explaining when we'd add this back in, we can remove this commented out line along with lines 43-44?